src/Microsoft.Azure.WebJobs.Host/Bindings/Path/BindingParameterResolver.cs

```diff
@@ -107,16 +108,8 @@ public override string Name
             public override string Resolve(string value)
             {
                 string format = GetFormatOrNull(value);
-
-                if (!string.IsNullOrEmpty(format))
-                {
-                    return DateTime.UtcNow.ToString(format, CultureInfo.InvariantCulture);
-                }
-                else
-                {
-                    // default to ISO 8601
-                    return DateTime.UtcNow.ToString("yyyy-MM-ddTHH-mm-ssK", CultureInfo.InvariantCulture);
-                }
+                var val = new SystemBindingData().UtcNow;
+                return BindingDataPathHelper.ConvertParameterValueToString(val, format);
```

Comment:
Discussed in person:
- we already have a centralized formatter.  This is just adding DateTime/Guid to it. This is actually a good thing since it enforces consistency 
- SystemBindingData does not do any formatting. It just exposes the builtin contract in one place.